enterprise/releases/2023-07-0.mdx

```diff
@@ -0,0 +1,80 @@
+
+## Features
+
+* Index Advisor
+  - Support analyzing queries that require the "ltree" extension <!-- PR 2913 -->
+  - Add support for IMMUTABLE functions <!-- PR 2937 -->
+    - This fixes errors like "functions in index expression must be marked IMMUTABLE" or "functions in partition key expression must be marked IMMUTABLE".
+  - Fix query analysis of INSERT INTO SELECT by ensuring correct column order for query analysis <!-- PR 2916 -->
+    - This fixes type coercion errors like "column "X" is of type A but expression is of type B" on specific cases of INSERT INTO SELECT statements that relied on the column positions of the target table.
+  - Support foreign key constraints that reference tables with indexes using custom functions <!-- PR 2915 -->
+  - Fix some cases of query analysis error "view depth limit (10) exceeded" <!-- PR 2900 -->
+  - Support schemas that require custom functions in check constraints/domain types <!-- PR 2968 -->
+  - Fix support for handling functions that call citext input/output functions <!-- PR 2964 -->
+* Connection Tracing: Increase Connection Traces live refresh timeout from 1 to 5 minutes <!-- PR 2892 -->
+* Check-Up: Show full query text instead of truncated one-line query for issue pages of Active Queries and Blocking Queries checks <!-- PR 2888 -->
+* Query details page: Update query text unavailable error message to explain how to fix the problem queries <!-- PR 2932 -->
+* Update bundled collector version to release 0.50.1 <!-- PR 2985 -->
+  - Log Insights: Add autovacuum index statistics information introduced in Postgres 14
+    - Previously, if autovacuum logs included such information, the collector failed to match the log line and the events would not be classified correctly in Log Insights
+  - In case you have installed the collector separately, we recommend upgrading the collector package/container and reviewing the [full changelog](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md#0501------2023-06-29).
+
+
+## Performance
+
+* Snapshot processing
+  - Improve speed of lock graph analysis for blocking queries alert <!-- PR 2987 -->
+  - Re-introduce limit of length on query parsing for very large queries (more than 200,000 characters) <!-- PR 2956 -->
+    - After the removal of the old 20,000 limit in the prior Enterprise release (v2023.06.0) we've received reports of this causing crashes
+    - In our testing the problem case here is when queries are extremely long (sometimes query texts exceeding 1MB in length) - this release re-introduces a limit of 200,000 (10x the old limit), which allows most cases but avoids the extreme cases that cause the snapshot worker to crash
+    - Queries exceeding this length limit will not be considered for Index Advisor, and will show under the "Other" statement type in the Query Performance view
```

Comment:
This one is confusing. What do you mean by "will not be considered for Index Advisor"? Do you mean "will not be analyzed by Index Advisor"?

Reply:
Yep, that's what I meant - will reword to your suggestion.